mira/dkg/viz.py

```diff
@@ -28,4 +29,10 @@ def draw_relations(records, fname, is_full=False):
                            label=relation['predicate'],
                            color="red", weight=2)
     agraph = nx.nx_agraph.to_agraph(graph)
-    agraph.draw(path=fname, prog="dot", format="png")
+    try:
+        agraph.draw(path=fname, prog="dot", format="png")
+    except FileNotFoundError:
+        click.echo("You passed in a fname that contains a forward slash that doesn't delineate"
```

Comment:
Let's not do error handling here. First, the error message is too specific since FileNotFoundError can happen for all sorts of reasons, not just due to some specific issue with forward slashes. But more broadly, we just don't need to catch errors here, any errors associated with drawing and saving the file can just be thrown as is.